docs/_toc.yaml

```diff
@@ -0,0 +1,29 @@
+toc:
+  - heading: Shardy documentation
+  - title: Getting started
+    section:
+    - title: Overview
+      path: overview.md
+    - title: Use Shardy from JAX
+      path: getting_started_jax.md
+    # - title: Use Shardy with MyDialect
+    #   path: getting_started_mydialect.md
+    # - title: Automatic partitioner
+    #   path: automatic_partitioner.md
```

Comment:
These pages are planned but not yet done. I included them here to highlight the proposed documentation structure.

Reply:
toc lgtm!

Reply:
Format update, looks like these files are super sensitive to whitespace:

```
toc:
- heading: Shardy documentation
- title: Getting started
  section:
  - title: Overview
    path: overview.md
  - title: Use Shardy from JAX
    path: getting_started_jax.md
  # - title: Use Shardy with MyDialect
  #   path: getting_started_mydialect.md
  # - title: Automatic partitioner
  #   path: automatic_partitioner.md
- title: Concepts
  section:
  - title: Shardy architecture
    path: architecture.md
  - title: Sharding representation
    path: sharding_representation.md
  - title: Propagation
    path: propagation.md
  - title: Compiler API
    path: compiler_api.md
- title: Reference documentation
  section:
  - sdy_dialect.md
  - sdy_export_passes.md
  - sdy_import_passes.md
  - sdy_op_interfaces.md
  - sdy_propagation_passes.md
```

Reply:
Also I'm posting some basic scaffolding to get the shardy site off the ground, you'll likely hit a merge conflict on this file, apologies. Just overwrite with your changes.

Also! Editing my previous toc file, all links need to be prefaced with `/shardy/mypage`, and the intro page (`index.html` equivalent) should be `/shardy`

Reply:
Shardy doc page is live, will pull in your updates within 24hr after submit (daily cron job):

https://openxla.org/shardy

Reply:
Thanks!